tested/languages/haskell/generators.py

```diff
@@ -48,37 +48,37 @@ def convert_value(value: Value) -> str:
         return f"({convert_arguments(value.data)})"
```

Comment:
This is where I would remove the typing information.

But i am not an expert on TESTed nor haskell so I am not sure it will work